src/main/java/seedu/address/model/task/Task.java

```diff
@@ -47,23 +51,64 @@ public Task(String taskName, LocalDateTime deadline, Boolean isDone) {
         name = taskName;
         this.deadline = deadline;
         this.isDone = isDone;
+        this.employee = new ArrayList<>();
     }
 
     /**
-     * Returns true if a given string is a valid project name.
+     * Constructs a Task with an assigned employee.
+     * - for loading of tasks from taskhub.json
+     * @param taskName The name of the task.
+     * @param employee The employee that was assigned to the task.
      */
-    public static boolean isValidTask(String testString) {
-        return testString.matches(VALIDATION_REGEX);
+    public Task(String taskName, LocalDateTime deadline, Employee employee) {
+        requireNonNull(taskName);
+        requireNonNull(deadline);
+        requireNonNull(employee);
```

Comment:
There is a 'requireAllNonNull` method, which will shorten the lines here from 3 to 1. Likewise for the other constructor methods in this class. I believe you're aware of this though, since you used the method in another class, so is there a reason for separating it here?

Reply:
No reason, I must have missed this. Thanks!